instrumentation/logback/logback-mdc-1.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/logback/mdc/v1_0/LoggingEventInstrumentation.java

```diff
@@ -79,6 +87,16 @@ public static void onExit(
       spanContextData.put(SPAN_ID, spanContext.getSpanId());
       spanContextData.put(TRACE_FLAGS, spanContext.getTraceFlags().asHex());
 
+      if (addBaggage) {
+        Baggage baggage = Java8BytecodeBridge.baggageFromContext(context);
```

Comment:
In theory at least, I think you can have baggage without a span, which wouldn't get here b/c of return on line 82 (and probably a good test case to add)